### ISSUE TITLE
Model issue

### ISSUE THREAD

**Issue by YuecongWang** [lghupan/Starfish#3]
Author, I would like to know about model_0.9797.h5 and how you trained it, because the original text does not have any origin about it, but it plays a large role in the loss of training

**Comment by lghupan** [comment]
Thank you for your interest in our work. Unfortunately, I am unable to locate the files that generate this classification model, but it should be straightforward to implement. Would you mind training your own model? I'm happy to help if you encounter any discrepancies in the results.

**Comment by YuecongWang** [comment]
Our professor thinks your work is very good, so I want to reproduce it and modify it, I think the form of the model should be resnet50, but the final dense layer has changed from 1000 to 196, I just wonder if it is set up specifically for car or just like the official weight in tensorflow, it is divided into 1000 classes

**Comment by YuecongWang** [comment]
I'm just a little skeptical if your weight has some self-defined properties, such as whether your model can be used for images of any other class, rather than a car and a class other than the class you defined to provide reproduction

**Comment by lghupan** [comment]
1) It is set up specifically for the cars dataset; the final dense layer has changed from 1000 to 196 as the dataset has 196 classes. 2) there are no self-defined properties in the weights.

After digging into some old documents, I was able to find some draft code from Junha on training the classification model. It is not the exact code that generates model_0.9797.h5, but I feel that it should be close.

[2_stanford_cars_lr_tuning.ipynb.zip](https://github.com/user-attachments/files/17338193/2_stanford_cars_lr_tuning.ipynb.zip)

**Comment by YuecongWang** [comment]
Thank you very much for your answers
